src/urls.cc

```diff
@@ -35,7 +35,7 @@ std::string API() {
 
 std::string TimelineUpload() {
     if (use_staging_as_backend) {
-        return "https://toggl.space";
+        return "https://timeline.toggl.space";
```

Comment:
I intentionally fix incorrect Timeline Endpoint for staging.

## Problem
- Access to Timeline Staging (https://toggl.space), but we get redirect to the Production -> Cause the error when updating Timeline Setting

<img width="1840" alt="Screen_Shot_2020-03-09_at_14_19_54" src="https://user-images.githubusercontent.com/5878421/76191382-d5d9ac80-6211-11ea-8f25-3645beab901e.png">

Reply:
👍